charts/abacus/templates/configmap.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "abacus.name" . }}-cm
+data:
+  KEYCLOAK_HOST: {{ .Values.oidc.serverUrl }}
+  KEYCLOAK_CLIENT_ID: {{ .Values.oidc.clientId }}
+  KEYCLOAK_REALM: {{ .Values.oidc.realm }}
```

```suggestion
  KEYCLOAK_REALM: {{ .Values.oidc.realm }}
  KEYCLOAK_REALMS: {{ .Values.oidc.queryRealms }}
```

There is also a `KEYCLOAK_REALMS` env var that Abacus supports that defines what Keycloak Realms you can actually access and log into. It can be set to comma-delimited realm names, this is what I have in Cleanroom: `"tdf,tdf-pki,dcr,dcr-pki"`

I do not know why there is a REALM vs REALMS, but I do know I have to set the latter to access more than one realm.

---

@obidenko ?

---

We dont need realm variable, only realms